### PR TITLE
Update GCP and AWS Quickstarts to no-fork providers

### PR DESCRIPTION
**Pull request by plumbis**
Updates the AWS quickstart to v0.46.0 and GCP to v0.40.0.

### CODE REVIEW

File: content/v1.14/getting-started/provider-aws.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Every CRD maps to a unique AWS service Crossplane can provision and manage.
 
 {{< hint type="tip" >}}
 See details about all the supported CRDs in the 
-[Upbound Marketplace](https://marketplace.upbound.io/providers/upbound/provider-aws-s3/v0.37.0).
+[Upbound Marketplace](https://marketplace.upbound.io/providers/upbound/provider-aws-s3/v0.46.0).
```
Comment (jeanduplessis) [inline content/v1.14/getting-started/provider-aws.md:70]:
```suggestion
[Upbound Marketplace](https://marketplace.upbound.io/providers/upbound/provider-aws-s3/v0.47.0).
```

File: content/v1.14/getting-started/provider-gcp-part-2.md [hunk 2]
```diff
@@ -47,7 +45,7 @@ kind: Provider
 metadata:
   name: provider-gcp-storage
 spec:
-  package: xpkg.upbound.io/upbound/provider-gcp-storage:v0.35.0
+  package: xpkg.upbound.io/upbound/provider-gcp-storage:v0.40.0
```
Comment (jeanduplessis) [inline content/v1.14/getting-started/provider-gcp-part-2.md:48]:
```suggestion
  package: xpkg.upbound.io/upbound/provider-gcp-storage:v0.41.0
```

File: content/v1.14/getting-started/provider-gcp-part-2.md [hunk 3]
```diff
@@ -99,6 +97,36 @@ EOF
...
+Add the new Provider to the cluster. 
+
+```yaml
+cat <<EOF | kubectl apply -f -
+apiVersion: pkg.crossplane.io/v1
+kind: Provider
+metadata:
+  name: provider-gcp-pubsub
+spec:
+  package: xpkg.upbound.io/upbound/provider-gcp-pubsub:v0.40.0
```
Comment (jeanduplessis) [inline content/v1.14/getting-started/provider-gcp-part-2.md:115]:
```suggestion
  package: xpkg.upbound.io/upbound/provider-gcp-pubsub:v0.41.0
```

File: content/v1.14/getting-started/provider-gcp-part-2.md [hunk 4]
```diff
@@ -99,6 +97,36 @@ EOF
...
+```
+
+View the new PubSub provider with `kubectl get providers`.
+
+```shell {copy-lines="1"}
+kubectl get providers
+NAME                          INSTALLED   HEALTHY   PACKAGE                                                AGE
+provider-gcp-pubsub           True        True      xpkg.upbound.io/upbound/provider-gcp-pubsub:v0.40.0    39s
+provider-gcp-storage          True        True      xpkg.upbound.io/upbound/provider-gcp-storage:v0.40.0   13m
+upbound-provider-family-gcp   True        True      xpkg.upbound.io/upbound/provider-family-gcp:v0.40.0    12m
```
Comment (jeanduplessis) [inline content/v1.14/getting-started/provider-gcp-part-2.md:126]:
```suggestion
provider-gcp-pubsub           True        True      xpkg.upbound.io/upbound/provider-gcp-pubsub:v0.41.0    39s
provider-gcp-storage          True        True      xpkg.upbound.io/upbound/provider-gcp-storage:v0.41.0   13m
upbound-provider-family-gcp   True        True      xpkg.upbound.io/upbound/provider-family-gcp:v0.41.0    12m
```

File: content/v1.14/getting-started/provider-gcp.md [hunk 5]
```diff
@@ -36,7 +36,7 @@ kind: Provider
 metadata:
   name: provider-gcp-storage
 spec:
-  package: xpkg.upbound.io/upbound/provider-gcp-storage:v0.35.0
+  package: xpkg.upbound.io/upbound/provider-gcp-storage:v0.40.0
```
Comment (jeanduplessis) [inline content/v1.14/getting-started/provider-gcp.md:39]:
```suggestion
  package: xpkg.upbound.io/upbound/provider-gcp-storage:v0.41.0
```

File: content/v1.14/getting-started/provider-gcp.md [hunk 6]
```diff
@@ -51,8 +51,8 @@ Verify the provider installed with `kubectl get providers`.
 ```shell {copy-lines="1",label="getProvider"}
 kubectl get providers
 NAME                          INSTALLED   HEALTHY   PACKAGE                                                AGE
-provider-gcp-storage          True        True      xpkg.upbound.io/upbound/provider-gcp-storage:v0.35.0   14m
-upbound-provider-family-gcp   True        True      xpkg.upbound.io/upbound/provider-family-gcp:v0.35.0    14m
+provider-gcp-storage          True        True      xpkg.upbound.io/upbound/provider-gcp-storage:v0.40.0   36s
+upbound-provider-family-gcp   True        True      xpkg.upbound.io/upbound/provider-family-gcp:v0.40.0    29s
```
Comment (jeanduplessis) [inline content/v1.14/getting-started/provider-gcp.md:55]:
```suggestion
provider-gcp-storage          True        True      xpkg.upbound.io/upbound/provider-gcp-storage:v0.41.0   36s
upbound-provider-family-gcp   True        True      xpkg.upbound.io/upbound/provider-family-gcp:v0.41.0    29s
```

**Review by jeanduplessis — CHANGES_REQUESTED** [review]

**Comment by jeanduplessis** [comment]
@plumbis bumped the version to the latest version in the suggestions.

Is it possible to control these with a variable in Hugo?

**Comment by plumbis** [comment]
We could but my concern is QA. I would rather have a slightly out of date provider version than have a user's first experience not work due to an oversight or behavior change on our part.

**Comment by jeanduplessis** [comment]
That's a reasonable consideration. We'll stick to manual updates until it becomes a real problem.

**Comment by plumbis** [comment]
Pushed a change with the provider version bump.

**Review by jeanduplessis — APPROVED** [review]

**Pull request merged**